aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteModelFieldTypeConverter.java

```diff
@@ -49,7 +49,7 @@
  * to <code>Model</code> properties and from <code>Model</code> properties to values that are
  * valid in a <code>SQLiteStatement</code>.
  */
-final class SQLiteModelFieldTypeConverter implements ModelFieldTypeConverter<Cursor, Model> {
+public final class SQLiteModelFieldTypeConverter implements ModelFieldTypeConverter<Cursor, Model> {
```

Comment:
Does this need to be public?

Reply:
unless I start to move stuff around, yes... There are two packages right now and some `public` members because those two packages share code. Cleaning this up would require a deeper refactor.